### PR TITLE
Build rubygem-puma with openssl support for smart-proxy

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: packages/foreman/rubygem-puma/rubygem-puma.spec [hunk 1]
```diff
@@ -23,6 +23,7 @@ Requires: %{?scl_prefix}rubygem(nio4r) < 3
 BuildRequires: %{?scl_prefix_ruby}ruby(release)
 BuildRequires: %{?scl_prefix_ruby}ruby-devel >= 2.2
 BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: openssl-devel
```
Comment (ekohl) [inline packages/foreman/rubygem-puma/rubygem-puma.spec:26]: Could you move this below the `# end` line so it doesn't get cleared on the next update?

Comment (ehelms) [reply packages/foreman/rubygem-puma/rubygem-puma.spec]: Done.

**Comment by ehelms** [comment]
Ooops, removed now

**Comment by ekohl** [comment]
Looks like it's now linked to openssl:
```
Requires: /bin/bash /bin/sh /opt/rh/rh-ruby25/root/usr/bin/ruby /usr/bin/env libc.so.6()(64bit) libc.so.6(GLIBC_2.14)(64bit) libc.so.6(GLIBC_2.2.5)(64bit) libc.so.6(GLIBC_2.3.4)(64bit) libc.so.6(GLIBC_2.4)(64bit) libcrypt.so.1()(64bit) libcrypto.so.10()(64bit) libcrypto.so.10(OPENSSL_1.0.2)(64bit) libcrypto.so.10(libcrypto.so.10)(64bit) libdl.so.2()(64bit) libm.so.6()(64bit) libpthread.so.0()(64bit) libpthread.so.0(GLIBC_2.2.5)(64bit) libruby.so.2.5()(64bit) libssl.so.10()(64bit) libssl.so.10(libssl.so.10)(64bit) rtld(GNU_HASH) tfm-runtime
```

**Pull request merged**